content/en/docs/ambient/usage/policy/index.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Policy enforcement in ambient mode
```

Comment:
Is this only Authz policy?

Reply:
I would like to see this under a page for authz policy vs under usage/policy as if folks don't need authz policy, they don't need to care about this. Or if folks don't need L4 authz policy, they don't need to worry about this.

Reply:
My suggestion is:

- this becomes a "Designing authorization policy for ambient mode" guide (or whatever other types of policy are different between L4 and L7 and need consideration)
- it is weighted to come after the initial L4/L7 feature introductions
- most of the content is here, and the L4/L7 feature introductions link out to this doc.

Reply:
ok, I thought we were attempting to get rid of a dedicated auth pol page for ambient. I'm ok with having it though.

Reply:
We got rid of a dedicated page to move policy into the journey of an adopter:

- here's how you install
- here's how you can do L4 things
- here's how you can add on L7 things

We didn't want to leave policy to after you've learned about L7.
However, a page on how you should think about and design policy is a good thing to have, and by the time you get to reading that, you should have understood L4/L7/etc architecturally.